code/__DEFINES/say.dm

```diff
@@ -41,7 +41,9 @@
 #define MODE_VOCALCORDS "cords"
 #define MODE_KEY_VOCALCORDS "x"
 
-#define MODE_SING "%"
+#define MODE_SING "sing"
```

Comment:
why is this changed? Does that mean you can't say "sing" without singing?

Reply:
we went over this when i made the original pr. mode != key. its a code only thing to make its purpose clear.

Reply:
it might come as a surpise but I do not remember every answer to my every comment ever